packages-next/types/src/config/index.ts

```diff
@@ -161,6 +162,24 @@ export type ImagesConfig = {
   };
 };
 
+// config.files
+export type FilesConfig = {
+  upload: FileMode;
+  maxSize?: number;
```

Comment:
Could you look into how the upload size limit works in graphql-upload compared with with this, I feel like we might want to align across all things that use the `Upload` scalar?